desktop-app/app/utils/searchUrlUtils.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Can you please combine this and `urlSearchResultSettings` into one module at `app/services/url-suggestion/index.js` ?